src/components/Events/FeedEvent/index.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from "react";
+import moment from "moment";
+import Linkify from "react-linkify";
+
+import "./style.css";
+
+const HIDE_PAST_EVENTS = true; // if true, only events that occur after today will be shown
+
+export default class FeedEvent extends Component {
+	
+	render() {
+
+		// If the event property does not exist
+		// do not display anything
+		let event = this.props.event;
+		if (!event) {
+			return null;
+		}
+
+		//let eventPlace = event.place;
+		if (HIDE_PAST_EVENTS) {
+			if (moment(event.start_time).diff(moment()) < 0) {
```

Comment:
This will require a correction on the Go server as well. I like this change though.